aconcli/cmd/kill.go

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"aconcli/service"
+	"github.com/spf13/cobra"
+)
+
+var killCmd = &cobra.Command{
+	Use:   "kill SIGNAL_NUM",
```

Comment:
Include and explain the options (`-c` and `-e`?) in the help messages.

Reply:
The descriptions for the flags will be displayed when invoking `aconcli kill -h`